provisioning/transport/http/src/Generated/Models/TpmAttestation.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices.Provisioning.Client.Transport.Http.Models
```

Comment:
>Http [](start = 64, length = 4)

should the attestation class be under non transport specific namespace?

Reply:
This is HTTP/JSON specific and so AMQP/MQTT shouldn't require this.